_includes/components/children_nav.html

```diff
@@ -0,0 +1,9 @@
+<hr>
+{% include toc_heading_custom.html %}
+<ul>
+  {% for child in toc_list %}
```

Comment:
The dependence on `toc_list` could (and perhaps should?) be made explicit by passing it as a [parameter](https://jekyllrb.com/docs/includes/#passing-parameters-to-includes) when including `children_nav.html`.

Reply:
Agreed; I'm just now realizing how pernicious the leakage between `nav.html` and the rest of the site is. I'll have to give how to solve this in a bigger picture more thought.